scripts/microshift-disable.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+SCRIPT_NAME=$(basename $0)
+
+# Exit if the current user is not 'root'
+if [ $(id -u) -ne 0 ] ; then
+    echo "The '${SCRIPT_NAME}' script must be run with the 'root' user privileges"
+    exit 1
+fi
+
+echo "Stopping MicroShift..."
+systemctl stop microshift
+
+echo "Stopping pods managed by kubelet..."
+systemctl stop kubelet.slice
+
+echo "Removing OVS configuration..."
+/usr/bin/configure-ovs.sh OpenShiftSDN
```

Comment:
There is a partial cleanup issue when `br-int` is removed in this script, details captured in this ticket: https://issues.redhat.com/browse/OCPBUGS-7778. This could result in pod fails to start after restart microshift service.
To fix the partial cleanup issue, we could either:
1. require a node reboot after running this script or
2. add an parameter in configure-ovs.sh to skip `br-int` bridge removal. for example: `/usr/bin/configure-ovs.sh disable`

Reply:
The reboot option is probably the safest. That would bring everything up freshly with the new IP settings. I'll add that to the document.

Reply:
The problem is that deleting the `br-int` interface may cut off the network connectivity. This is what happens when we're uninstalling MicroShift RPMs.

Reply:
@ggiguash Yes, deleting br-int interface will disconnect pod network, good capture on that!
We could either reboot the node to fix it or not deleting br-int in this script.

Reply:
The reboot can only be performed from the current script to make sure the operation is atomic and network is back. 
However, since we're trying to temporarily "undo" OVN configuration for updating the IP address from cockpit, this creates a race with the reboot. 
Looks like we need to add the new IP setting + reboot to the same script to have everything consistent. Any other ideas?

Reply:
My goal isn't to combine everything into 1 script, it's to have a process they can follow.

What network connectivity is cut off when `br-int` is removed? I didn't see a problem using a browser outside of the VM to access cockpit on the VM. In fact, the cockpit UI updated live as the script ran.

Reply:
External network connectivity may be disrupted when switching between SDN and OVN modes if there is no reboot. Until now, this happened when uninstalling and reinstalling MicroShift. With this script, we're adding one more similar flow.